public/style/styles-compiler.scss

```diff
@@ -1,539 +1,18 @@
-}
-
-@keyframes rain {
-  0% {
-    opacity: 0;
-    transform: translateY(-.1em);
-  }
-  70% {
-    opacity: 1;
-  }
-
-  90% {
-    opacity: .9;
-  }
-
-  100% {
-    transform: translateY(0);
-    opacity: 0;
-  }
-}
-
-@keyframes bolt-flash {
-  0% {
-    opacity: 1;
-    transform: translateY(-.2em);
-  }
-
-  20% {
-    opacity: 0;
-  }
-
-  40% {
-    opacity: 1;
-    transform: translateY(0em);
-  }
-
-
-  100% {
-    opacity: 0;
-  }
-}
-
-// .fade-in {
-//     animation: fade-in .5s ease-in-out;
-// }
-
-@keyframes fade-in {
-  0% {
-    transform: scale3d(0, 0, 0);
-    opacity: 0;
-  }
-
-  50% {
-    transform: scale3d(1.05, 1.05, 1.05);
-  }
-
-  100% {
-    opacity: 1;
-    transform: scale3d(1, 1, 1);
-  }
-}
-
-
-/* ---------------------------------------- */
-/* Utilities */
-
-// flex utilities
-
-.flex {
-  display: flex;
-  align-items: flex-start;
-}
-
-.f-1 {
-  flex: 1;
-}
-
-.wrap {
-  flex-wrap: wrap;
-}
-
-.center {
-  align-self: center;
-}
-
-.center-h {
-  justify-content: center;
-}
-
-// spacing utilties
-
-.p-a-1 {
-  padding: $unit-spacing;
-
-}
-
-.m-a-1 {
-  margin: $unit-spacing;
-}
-
-.m-x-1 {
-  margin-right: $unit-spacing;
-  margin-left: $unit-spacing;
-}
-
-.m-b-1 {
-  margin-bottom: $unit-spacing;
-}
-
-.m-b-xs {
-  margin-bottom: $unit-spacing / 2;
-}
-
-// 
-
-.nowrap {
-  white-space: nowrap;
-}
-
-/* ---------------------------------------- */
-/* SVG icon, colors */
-
-svg { // width: 1.5em inlined
-  @extend .center;
-  @extend .m-x-1;
-  min-width: 32px; // fallback
-  max-width: 72px; // fallback
-  min-width: 4rem;
-  max-width: 6rem;
-  max-height: 72px; // fallback
-  max-height: 6rem;
-  padding: .1em;
-  box-sizing: content-box;
-}
-
-.sun {
-  fill: #F5A623;
-}
-
-.moon {
-  fill: #1E256C;
-}
-
-.stars {
-  fill: $purple-fallback;
-  fill: $purple;
-}
-
-// cloudy & fog
-.cloud {
-  fill: #36383C;
-}
-
-// rain, drizzle, snow
-.rain {
-  fill: #2B598E;
-}
-
-// sleet, hail
-.hail {
-  fill: $blue-darker;
-}
-
-.wind {
-  fill: #6F8F9D;
-}
-
-.tornado {
-  fill: #003952;
-}
-
-.bolt {
-  fill: #F2E000;
-}
-
-/* ---------------------------------------- */
-/* // Accessibility */
-
-.sr-only {
-  position: absolute;
-  width: 1px;
-  height: 1px;
-  padding: 0;
-  margin: -1px;
-  overflow: hidden;
-  clip: rect(0,0,0,0);
-  border: 0;
-}
-
-/* rule sets for windows high contrast mode */
-@media (-ms-high-contrast: active) {
-  *, body {
-    background: #000 !important;
-    color: #fff !important;
-    font-size: 32px;
-    font-size: 2rem;
-  }
-  a, button, input {
-    color: #0ff !important; // cyan
-    text-decoration: underline !important;
-    text-shadow: none !important;
-  }
-  button, input {
-    border: 1px solid #0ff !important;
-  }
-  a:hover, a:focus, button:hover, button:focus {
-    color: #6F0 !important; // neon green
-  }
-  svg {
-    display: none;
-  }
-}
+@import 'variables';
+
+@import 'base/reset',
+'base/typography',
+'base/structure',
+'base/utilities',
```

Comment:
Maybe utilities and high contrast should be moved into an `overrides` folder or something?